api/test/wfl/integration/modules/covid_test.clj

```diff
@@ -119,6 +84,45 @@
     (is (not (:started workload)))
     (is (:started (workloads/start-workload! workload)))))
 
+(deftest test-update-terra-executor
+  (fixtures/with-temporary-workspace
+    workspace-prefix group
+    (fn [workspace]
+      (let [snapshot {:name "test-snapshot-name"
+                      :id   (str (UUID/randomUUID))}
+            source (make-queue-from-list [snapshot])
+            executor (jdbc/with-db-transaction [tx (postgres/wfl-db-config)]
+                       (->> {:name                       "Terra"
+                             :workspace                  workspace
+                             :methodConfiguration        "mc-namespace/mc-name"
+                             :methodConfigurationVersion 1
+                             :entity                     "snapshot"
```

Comment:
We don't need the entity anymore I don't think?

Reply:
You're right, I pulled it from the spike doc but we never use it, will clean up!